main/resource-access.bs

```diff
@@ -142,19 +139,151 @@ through POST, using the HTTP `Slug` header as defined in [[!RFC5023]].
 Clients who want the server to determine the URI for their resource, MUST use
 the POST request.
 
-Servers MUST allow creating new resources with a POST request to URI path
+[[Source](https://github.com/solid/specification/pull/160#issuecomment-636822687)].
+
+### Reading resources ### {#read}
+
+Servers MUST support the HTTP `GET`, `HEAD` and `OPTIONS` methods [[!RFC7231]]
+for clients to read resources or to determine communication options.
+[[Source](https://github.com/solid/specification/issues/39#issuecomment-538017667)]
+
+When using Web Access Control ([[#wac]]):
+
+Servers MUST indicate their support for HTTP Methods by responding to HTTP
+`GET`, `HEAD` and `OPTIONS` requests on the resource's URL with the HTTP
+Method tokens in the HTTP response header `Allow`.
+
+Servers MUST indicate supported media types in the `Accept-Patch` [[!RFC5789]]
+and the `Accept-Post` [[!LDP]] headers for matching HTTP methods in `Allow`
+header value.
+
+Servers MAY include the HTTP `Accept-Patch` and `Accept-Post` headers in the
+response of a `OPTIONS *` request.
```

Comment:
Can likely be removed if my above 2 questions are answered.

Reply:
I suppose it can be removed. @kjetil raised the point that this should be said so that Solid implementations shouldn't spend time on it.

Reply:
OK, I did miss that it was `OPTIONS *` (so STAR), but perhaps we can cover that in one go above.